src/vaillant_netatmo_api/thermostat.py

```diff
@@ -632,11 +640,20 @@ class MeasurementType(Enum):
     SETPOINT_TEMPERATURE = "sp_temperature"
     SUM_BOILER_ON = "sum_boiler_on"
     SUM_BOILER_OFF = "sum_boiler_off"
-
+    SUM_ENERGY_GAS_HEATING = "sum_energy_gaz_heating"
+    SUM_ENERGY_GAS_WATER = "sum_energy_gaz_hot_water"
+    SUM_ENERGY_ELEC_HEATING = "sum_energy_elec_heating"
+    SUM_ENERGY_ELEC_WATER = "sum_energy_elec_hot_water"    
 
 class MeasurementScale(Enum):
     """MeasurementScale enumeration representing possible scale options for measurements of the thermostat."""
 
     MAX = "max"
+    MIN_5 = "5min"
     HALF_HOUR = "30min"
     HOUR = "1hour"
+    HOURS_3 = "3hours"
+    HOURS_6 = "6hours"
+    DAY = "1day"
+    WEEK = "1week"
+    MONTH = "1month"
```

Comment:
I'm thinking it might look nicer if we'd use full names, as that's consistent with the `HALF_HOUR` name.

```suggestion
    MAX = "max"
    FIVE_MINS = "5min"
    HALF_HOUR = "30min"
    HOUR = "1hour"
    THREE_HOURS = "3hours"
    SIX_HOURS = "6hours"
    DAY = "1day"
    WEEK = "1week"
    MONTH = "1month"
```